packages/web-react/src/types/pill.ts

```diff
@@ -1,7 +1,7 @@
 import { ElementType, JSXElementConstructor } from 'react';
 import { ActionColor, ChildrenProps, EmotionColor, TransferProps } from './shared';
 
-export type PillColor = 'selected' | 'unselected';
+export type PillColor<P> = ActionColor<P> | EmotionColor<P> | 'selected' | 'unselected';
```

Comment:
We want to only extend the Pill colors, not the dictionaries themselves.
```suggestion
export type PillColor<C> = ActionColor<void> | EmotionColor<void> | 'selected' | 'unselected' | C;
```